server/oam/kubernetes/workloads/v1.25.0/Secret_schema.k8s.json

```diff
@@ -1 +1,42 @@
-{"$schema":"http://json-schema.org/draft-07/schema","description":"Secret holds secret data of a certain type. The total bytes of the values in the Data field must be less than MaxSecretSize bytes.","properties":{"data":{"additionalProperties":{"format":"byte","type":"string"},"description":"Data contains the secret data. Each key must consist of alphanumeric characters, '-', '_' or '.'. The serialized form of the secret data is a base64 encoded string, representing the arbitrary (possibly non-string) data value here. Described in https://tools.ietf.org/html/rfc4648#section-4","type":"object"},"immutable":{"description":"Immutable, if set to true, ensures that data stored in the Secret cannot be updated (only object metadata can be modified). If not set to true, the field can be modified at any time. Defaulted to nil.","type":"boolean"},"stringData":{"additionalProperties":{"type":"string"},"description":"stringData allows specifying non-binary secret data in string form. It is provided as a write-only input field for convenience. All keys and values are merged into the data field on write, overwriting any existing values. The stringData field is never output when reading from the API.","type":"object"},"type":{"description":"Used to facilitate programmatic handling of secret data. More info: https://kubernetes.io/docs/concepts/configuration/secret/#secret-types","type":"string"}},"title":"Secret","type":"object","x-kubernetes-group-version-kind":[{"group":"","kind":"Secret","version":"v1"}]}
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
merge conflicts?

Reply:
@Abhishek-kumar09 I will fix this today. Please add your field addition in this PR so that I can test deployment before merge.